build/helper/metadata_find.py

```diff
@@ -5,17 +5,17 @@ def find_parameter(name, parameters):
     return parameter[0]
 
 
-def find_size_parameter(parameter_list, parameters):
+def find_size_parameter(parameter_list, parameters, key='value'):
     '''Returns the parameter that is used to specify the size other parameters. Applies to 'ivi-dance' and 'passed-in'.'''
```

Comment:
Does this apply to `ivi-dance-with-a-twist`? update comment.

Why is the key different in some cases? It'd be nice if it didn't need to be the case.